File: app/src/main/java/net/frju/flym/ui/main/FeedGroup.java
package net.frju.flym.ui.main;

import com.thoughtbot.expandablerecyclerview.models.ExpandableGroup;

import net.frju.flym.data.Feed;

import java.util.List;


public class FeedGroup extends ExpandableGroup<Feed> {

    public FeedGroup(String title, List<Feed> items) {
        super(title, items);
    }
}

File: app/src/main/java/net/frju/flym/ui/main/FeedGroup.kt
package net.frju.flym.ui.main

import com.bignerdranch.expandablerecyclerview.model.Parent
import net.frju.flym.data.entities.Feed


class FeedGroup(val feed: Feed, val subFeeds: List<Feed>) : Parent<Feed> {

    override fun getChildList(): List<Feed> {
        return subFeeds
    }

    override fun isInitiallyExpanded(): Boolean {
        return false
    }
}
